### DISCUSSION TITLE
How can I unlock a user?

### DISCUSSION THREAD

**Discussion by dacre-riv** [gravitational/teleport#6494] [Q&A] [answered]
I have a user with a status of 'is_locked: true' that has not changed (long after the lock_expires time has passed).  Is there a way to override this unwanted lock?

**Comment by benarent** [comment, marked as answer]
Hi @dacre-riv , 

This is an easy fix, on the Auth server you want to use https://goteleport.com/docs/cli-docs/#tctl-get `tctl get`. e.g. `tctl get user/joe > joe.yaml` Edit `joe.yaml` to remove is locked: false, then update Joe using `tctl update joe.yaml`
